src/App.jsx

```diff
@@ -1,12 +1,13 @@
-import './App.css';
-import Header from './Header';
-
-import UserData from './components/UserData';
+/*import "./App.css";*/
```

Comment:
At some point we will want to remove this line but it's fine to merge with it for now.

Reply:
I wasn't sure if I could delete this file due to the recent situation, so I left it as a comment